public/index.php

```diff
@@ -2,25 +2,18 @@
 
 declare(strict_types=1);
 
-require_once '../src/Car.php';
+spl_autoload_register(function ($class){
+    $class = __DIR__ . '/../' .lcfirst(str_replace('\\','/', $class)) .'.php';
+    require_once $class;
+    var_dump($class);
```

Comment:
1. var_dump nevajag atstāt.
2. $class nav nepieciešams uzstādīt atsevišķā variablē, to var returnot uzreiz